### PR TITLE
bringup ska95

### PR DESCRIPTION
**Pull request by spolack**

### CODE REVIEW

File: locations/ska95.yml [hunk 1]
```diff
@@ -48,6 +53,11 @@ networks:
     assignments:
       ska95-core: 1
 
+  - vid: 41 # Private LAN where private Internet is connected
+    role: ext
```
Comment (pktpls) [inline locations/ska95.yml:58]: This is called `uplink` now

**Comment by Noki** [comment]
The geolocation is wrong. Please have a look at hopglass.

**Comment by pktpls** [comment]
And I think some IPs are mixed up, or is it indeed 10.31.8.x *and* 10.36.8.x?

File: locations/ska95.yml [hunk 2]
```diff
@@ -0,0 +1,77 @@
+---
+
+location: ska95
+location_nice: Skaliter Str. 78
+latitude: 52.52075
+longitude: 13.40948
```
Comment (Noki) [inline locations/ska95.yml:6]: should be 52.500043, 13.43242

File: locations/ska95.yml (hunk 1 above)
Comment (spolack) [reply locations/ska95.yml:58]: But then i'd question the name. why should it be uplink when there is no uplink?

File: locations/ska95.yml (hunk 2 above)
Comment (spolack) [reply locations/ska95.yml]: thx!

File: locations/ska95.yml (hunk 1 above)
Comment (pktpls) [reply locations/ska95.yml:58]: I always thought the purpose of the `ext` role was to attach an uplink network, usually DSL or mobile. I've never seen it used without a tunnel

Comment (spolack) [reply locations/ska95.yml:58]: i was probably the only one who ever has used it. :)

Comment (pktpls) [reply locations/ska95.yml:58]: It's a legit use case, it's just news to me :-)

**Comment by Noki** [comment]
I fixed the typo in the street name and adjusted the location based on flb2net's input.

File: locations/ska95.yml (hunk 1 above)
Comment (Noki) [reply locations/ska95.yml:58]: ok, so the ext changes / fixes are fine? can we merge this?

**Pull request merged**